### PR TITLE
Add bean-format

### PR DESCRIPTION
**Pull request by elken**
Might also require some kind of deindent formatter (may be that we
introduce some generic formatters like that to chain other against?)
but otherwise should work fine

### CODE REVIEW

**Comment by elken** [comment]
Test will likely fail due to #99

File: test/formatters/installers/bean-format.bash [hunk 1]
```diff
@@ -0,0 +1,2 @@
+apt install -y python3-pip
```
Comment (raxod502) [inline test/formatters/installers/bean-format.bash:1]:
```suggestion
apt-get install -y python3-pip
```

File: test/formatters/installers/bean-format.bash [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+pip install beancount
```
Comment (raxod502) [inline test/formatters/installers/bean-format.bash:2]:
```suggestion
pip3 install beancount
```

**Comment by raxod502** [comment]
Interesting, what's the indentation problem? Does the formatter not work well when applied to an already-formatted file?

**Comment by elken** [comment]
It's just a very simple formatter that doesn't deindent, even the tests just use python `textwrap` to do that.

**Comment by raxod502** [comment]
What do you mean by `deindent`? When I hear that word I think of reducing the indentation level of a block of code, but your reference to the `textwrap` module makes me think you're talking about hard-wrapping long lines of text into paragraphs.

**Comment by elken** [comment]
I do indeed mean reducing indentation, perhaps the tests are misusing that module then.

All this does is align columns of numbers, if the indentation is wonky it doesn't adjust it.

**Comment by raxod502** [comment]
I think it's fine to leave as is. If formatting this file format requires additional functionality then it should be added into the formatter, not hacked on top in an Emacs package!

**Pull request merged**